Task1_EX_St.py

```diff
@@ -0,0 +1,43 @@
+class ATM:
+    def __init__(self, customer_balance, min_withdrawal_amount, max_withdrawal_amount):
+        self.customer_balance = customer_balance
+        self.min_withdrawal_amount = min_withdrawal_amount
+        self.max_withdrawal_amount = max_withdrawal_amount
+
+    def withdraw_cash(self, amount):
+        if amount < self.min_withdrawal_amount or amount > self.max_withdrawal_amount:
```

Comment:
for visual aid, it can be helpful to group the expressions around the or in brackets, for example if (amount < self.min_withdrawal_amount) or (....)